### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by JanpashaShaik12**

### CODE REVIEW

File: Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/ClientClass.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
+package tddmicroexercises.telemetrysystem;
+
+import java.util.Random;
+
+import static tddmicroexercises.telemetrysystem.TelemetryClient.DIAGNOSTIC_MESSAGE;
+
+public class ClientClass implements Client{
```
Comment (AbhilashZemoso) [inline Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/ClientClass.java]: Lets not prefix or suffix the class name with "Class"
Rename it

File: Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/ClientClass.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    private static boolean onlineStatus;
+    private static  String diagnosticMessageResult = "";
```
Comment (AbhilashZemoso) [inline Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/ClientClass.java]: remove static

File: Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/ConnectionClass.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package tddmicroexercises.telemetrysystem;
+import java.util.Random;
+
+public class ConnectionClass implements Connection{
```
Comment (AbhilashZemoso) [inline Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/ConnectionClass.java]: Lets not prefix or suffix the class name with "Class"
Rename it

File: Java/TelemetrySystem/src/test/java/tddmicroexercises/telemetrysystem/TelemetryDiagnosticControlsTest.java [hunk 4]
```diff
@@ -1,13 +1,35 @@
...
 
+        } catch (Exception e) {
+            Assert.assertNotEquals("Unknown exception", e);
+        }
+    }
+    @Test
+    void checkGetDiagnostic(){
+       String dignosticInfo =  telemetryDiagnosticControls.getDiagnosticInfo();
+       Assert.assertEquals(dignosticInfo,telemetryDiagnosticControls.getDiagnosticInfo());
+    }
```
Comment (AbhilashZemoso) [inline Java/TelemetrySystem/src/test/java/tddmicroexercises/telemetrysystem/TelemetryDiagnosticControlsTest.java]: this Test case is doesn't make sense
Remove it

**Review by AbhilashZemoso — CHANGES_REQUESTED** [review]
work on comments

File: Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/ClientClass.java (hunk 1 above)
Comment (JanpashaShaik12) [reply Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/ClientClass.java]: Renamed the classes

File: Java/TelemetrySystem/src/test/java/tddmicroexercises/telemetrysystem/TelemetryDiagnosticControlsTest.java (hunk 4 above)
Comment (JanpashaShaik12) [reply Java/TelemetrySystem/src/test/java/tddmicroexercises/telemetrysystem/TelemetryDiagnosticControlsTest.java]: Removed the Test case

File: Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/ConnectionClass.java (hunk 3 above)
Comment (JanpashaShaik12) [reply Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/ConnectionClass.java]: Renamed the class to COnnectionImpl

File: Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/ClientClass.java (hunk 2 above)
Comment (JanpashaShaik12) [reply Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/ClientClass.java]: Removed static

**Review by AbhilashZemoso — APPROVED** [review]
LGTM

**Pull request merged**